backend/app/tests/utils/api_keys.py

```
@@ -0,0 +1,30 @@
+import json
+from pathlib import Path
+from typing import Dict
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)